### ISSUE TITLE
English string and aria-labels in Danish Mozilla Accounts Settings Page

### ISSUE THREAD

**Issue by Joergen** [mozilla/fxa#18571]
## Description
English string and aria-labels in Danish Mozilla Accounts Settings Page.

## Steps to reproduce
Change your preferred language for displaying pages to Dansk/Danish in Firefox settings.
Open https://accounts.firefox.com/settings
Scroll down to the "Totrinsgodkendelse"/"Two-step authentication" section.

## Actual result
1. The button text "Create new codes" is in English.
2. Three aria-labels "Enabled" is in English.
3. Aria-label "Backup authentication codes enabled" is in English.

## Expected result
1. The button text should be "Opret nye koder".
2. The three aria-labels should be "Aktiveret".
3. The aria-label should be "Reserve-godkendelseskoder aktiveret".

### Environment

136.0.1 (64-bit) Linux Mint 22.1
![Danish_Mozilla_Accounts_Settings](https://github.com/user-attachments/assets/2840b987-c3de-4022-8714-e57b35f9c0e5)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXA-11374)

**Comment by flodolo** [comment]
Confirming for Italian, it's hard-coded
https://github.com/mozilla/fxa/blob/81affb46795004fed793df2186fb7846021e4a13/packages/fxa-settings/src/components/Settings/SubRow/index.tsx#L200

CC @bcolsson

**Comment by bcolsson** [comment]
I tested in storybook and in production and I'm seeing the button text display correctly:

![Image](https://github.com/user-attachments/assets/770fd1f4-4865-4720-8002-a9553b1c6254)

Note: @flodolo - I believe the code you showed is how fallbacking is handled in fxa.

**Comment by bcolsson** [comment]
However, I can confirm that the .aria-labels are not localized on the settings page.

@vpomerleau - it appears that many of the icons on the settings page don't have localizable .aria-labels. Some I'm seeing currently are  "Enabled" / "Alert" / "Backup authentication codes enabled" / "Recovery SMS Disabled"

**Comment by Joergen** [comment]
> I tested in storybook and in production and I'm seeing the button text display correctly:

The button text "Create new codes"/"Opret nye koder" is in Danish now.
